tests/drivers/watchdog/wdt_basic_api/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,3 @@
+&wdt30 {
```

Comment:
header missing here and some other places
when added it's sometimes inconsistent